### PR TITLE
Change beautifulsoup setting to "lxml-xml".

### PR DESCRIPTION
**Pull request by gnott**
If tests pass - this should remove the warning message.

### CODE REVIEW

File: elifetools/parseJATS.py [hunk 1]
```diff
@@ -12,7 +12,7 @@
 from utils import unicode_value
 
 def parse_xml(xml):
-    return BeautifulSoup(xml, ["lxml", "xml"])
+    return BeautifulSoup(xml, "lxml-xml")
```
Comment (giorgiosironi) [inline elifetools/parseJATS.py]: I'm only a casual user of `beautifulsoup`, I know this is the name of the parser, but couldn't find documentation that lists the possible values

Comment (giorgiosironi) [reply elifetools/parseJATS.py]: Tests seem to like it though

Comment (gnott) [reply elifetools/parseJATS.py]: There are some options listed at https://www.crummy.com/software/BeautifulSoup/bs4/doc/#installing-a-parser

I believe when we upgraded BeautifulSoup in the requirements.txt, this is the recommended value for that version, as opposed to the former value.

**Comment by lsh-0** [comment]
can this be merged @gnott ?

**Comment by gnott** [comment]
I think so, @giorgiosironi did not "approve" but I think his comment is addressed sufficiently. I will merge!

**Comment by lsh-0** [comment]
thanks @gnott

**Comment by lsh-0** [comment]
> giorgiosironi did not "approve"

I see code review as an opportunity for others to cast an opinion rather than give permission. I think you're fine to have merged.

**Comment by giorgiosironi** [comment]
Yes, I had no particular opinions or knowledge on the matter to add to the discussion

**Comment by lsh-0** [comment]
aw, but you know they're always welcome ;)

**Pull request merged**